### PR TITLE
Vault 1.5.9 Pre-staging

### PR DESCRIPTION
**Pull request by HridoyRoy**

### CODE REVIEW

File: sdk/version/version_base.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ var (
 	// Whether cgo is enabled or not; set at build time
 	CgoEnabled bool
 
-	Version           = "1.5.8"
-	VersionPrerelease = ""
+	Version           = "1.5.9"
+	VersionPrerelease = "rc"
```
Comment (vishalnayak) [inline sdk/version/version_base.go:12]: This is not an RC. Why is this set?

Comment (HridoyRoy) [reply sdk/version/version_base.go:12]: Oh, do minor releases not have release candidates? Should I just set it back to `""` ?

Comment (vishalnayak) [reply sdk/version/version_base.go:12]: I don't think so. In my opinion, yes, it should be set to "".

Comment (HridoyRoy) [reply sdk/version/version_base.go:12]: Sounds good!

**Review by vishalnayak — APPROVED** [review]

**Review by briankassouf — APPROVED** [review]

**Pull request merged**